### DISCUSSION TITLE
Use tag icon as browser icon

### DISCUSSION THREAD

**Discussion by webysther** [linuxserver/Heimdall#1095] [Ideas]
After creating a tag, using the icon as icon in browser is interesting to put in fav.

Sample:

Tag tech
![image](https://user-images.githubusercontent.com/750007/211783197-0e6cfa09-25d4-4d67-b6d1-3776b5dd4840.png)

When saved in browser shows heimdall icon, but i prefer the icon selected for the tag:

![image](https://user-images.githubusercontent.com/750007/211783397-afd53c1f-ee62-4530-a0be-170296427ea6.png)

**Comment by keriati** [comment]
It's an interesting idea, moved it to ideas.
